.github/workflows/backend-prod-cd.yml

```
@@ -79,233 +79,34 @@ jobs:
-            if [ $i -eq 10 ]; then
-              echo "서버가 정상적으로 실행되지 않아 종료함"
-              exit 1
-            fi
-          done
-
-          echo "-- 배포 종료 --"
-
-  deploy-app-server2:
-    name: Deploy to App Server 2
+    if: needs.detect-idle.outputs.idle-server == 'prod-app-1'
+    uses: .github/workflows/prod-app-1-cd.yml
```

actionlint (1.7.7)

reusable workflow call ".github/workflows/prod-app-1-cd.yml" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)